CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ Please avoid adding duplicate information across this changelog and JIRA/doc inp
   does not match the `:authority` (`Host`) header. This feature can be turned off by setting the environment variable
   `ROX_ALLOW_MISDIRECTED_REQUESTS=true`.
 - Fixed permissioms checks in the UI that prevented users with certain limited permissions from creating report configurations.
+- Registry integrations for ECR are now auto-generated if the cluster's cloud provider is AWS, and the nodes' Instance IAM Role has policies granting access to ECR.  Customers can turn this feature off by setting the environment variable `ROX_ECR_AUTO_INTEGRATION=false`, or by disabling the EC2 instance metadata service in their nodes.
```

Comment:
They actually cannot do that because this is a feature flag. In release builds of RHACS/StackRox, the default value for the feature flag is used, and it cannot be overridden. If you want a flag which users can turn on and off, then you'd have to make it an environment variable similar to something like what you'd find here: https://github.com/stackrox/stackrox/tree/master/pkg/env

Also, if users use the Operator, it may not be as easy to configure this setting. We'd have to probably add some additional installation configuration.

There is probably no reason why someone would want to disable this once we enable it, right? Is there any case where having this enabled breaks pre-existing integrations?

Reply:
Thank you for clarifying.

I don't know why someone would want this disabled other than preference. In particular, it should not have any impact on current integrations. However, I know some customers have complained about how they cannot delete auto-generated integrations. So I thought it would be interesting to see that they can disable auto-generated ECR integrations.

But obviously, I misunderstood how feature flags work.

So, I am OK leaving it as a feature flag, and I will update the changelog entry and remove the mention of how it can be enabled or disabled through an environment variable.